draft-homburg-deleg-incremental-deleg.md

```diff
@@ -768,7 +768,12 @@ The source code of his implementation is available on github {{DELEG4UNBOUND}}
 
 # Security Considerations
 
-TODO Security
+Incremental deleg moves the location of  referral information to a unique  location that currently exists. However, as this is a new approach, thought must be given to usage.   There must be some checks to ensure that the registering a _deleg subdomain happens at the time the domain is provisioned. The same care needs to be addressed when a domain is deprovisioned that the _deleg is removed.  This is similar to what happens to NS A records deployed in parent zones to act as Glue.
```

Comment:
```suggestion
Incremental deleg moves the location of  referral information to a unique  location that currently exists. However, as this is a new approach, thought must be given to usage.   In domains that are open to registration by third parties, care must be taken to ensure that the _deleg subdomain is reserved at the time the domain is provisioned. The same care needs to be addressed when a domain is deprovisioned that the _deleg reservation is removed.  This is similar to what happens to NS A records deployed in parent zones to act as Glue.
```

Reply:
I like this suggestion. Should we wait for @moonshiner to commit it?

Reply:
Yes (unless we get too close to midnight)

Reply:
COMMIT NOW